src/components/Chat/MessagesList/MessagesList.js

```diff
@@ -10,10 +10,17 @@ import { useSelector } from 'react-redux';
 import Message from '../Message';
 
 function MessagesList() {
-  const messages = useSelector((state) => state.messages);
   const wrapperRef = useRef(null);
   const messagesRef = useRef(null);
 
+  const filterMessages = (messages) => {
+    // Avoid the pointless initial stream of messages like "x has joined the room"
+    // when entering a room in which there are partitipants already. Even if that
+    // means filtering a bit too much in some cases.
+    const first = messages.findIndex((msg) => msg.type !== 'newRemoteFeed');
```

Comment:
Just for the record, I think this could be problematic when using the "show/hide chat" feature, but I did not test it yet. I'll do during the week.